### PR TITLE
Added initialize_2d_array and some

### PR DESCRIPTION
**Pull request by TheComputerM**
This adds snippets of `initialize_2d_array` and `some`. An advanced snippet and intermediate snippet respectively.

### CODE REVIEW

File: snippets/some.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: some
+tags: array,function,intermediate
+---
+
+Returns `True` if the provided function returns `True` for at least one element in the array, `False` otherwise.
```
Comment (logankilpatrick) [inline snippets/some.md]: True and False should be lowercase here.

**Comment by TheComputerM** [comment]
Will do the changes once I am on my laptop.

**Comment by TheComputerM** [comment]
@logankilpatrick  Fixed

**Comment by TheComputerM** [comment]
Added new logo as a part of GCI

**Comment by logankilpatrick** [comment]
Hey @TheComputerM Can you make the logo update in a different Pull Request?

**Comment by TheComputerM** [comment]
Ok will do

**Pull request closed without merging**